src/ecs/systems/client/SendDirection.hpp

```diff
@@ -0,0 +1,44 @@
+/*
+** EPITECH PROJECT, 2022
+** RTYPE
+** File description:
+** HandleIncomingMessages
+*/
+
+#pragma once
+
+#include <functional>
+#include <iostream>
+#include "../client/NetworkClient.hpp"
```

Comment:
i'm sure you can do better @Cavonstavant

Reply:
i'm on it @Bootoyka